### DISCUSSION TITLE
Touch mode - Hide mouse and scrolling

### DISCUSSION THREAD

**Discussion by Classical-Gas** [jlesage/docker-firefox#195] [Ideas]
Hi,

I'm using this docker image to breathe life in to an old table that can't run my native smart home dashboard it's its working very well!

My only wishes are to be able to hide the cursor input and allow for touch-screen style scrolling. Would it be possible to introduce this with an environmental variable?

Cheers,

**Comment by arunzn** [comment]
I would also love this feature

**Comment by jlesage** [comment]
> and allow for touch-screen style scrolling

What does this mean exactly ?

**Comment by arunzn** [comment]
I am trying to access certain websites that requires horizontal scrolling. Example is provided below. It is being access from mobile browser. It is impossible to do horizontal scroll naturally, but can do by dragging scroll bar. This is not intuitive at all. Its a very important use case for us.


docker run -d \
-p 5800:5800 \
--shm-size 2g \
-e DISPLAY_WIDTH=450 \
-e DISPLAY_HEIGHT=770 \
--name firefox \
-e FF_KIOSK=1 \
-e FF_OPEN_URL=http://www.thehorizontalway.com/ \
jlesage/firefox

**Reply by arunzn** [reply]
When I scroll in mobile phone the content gets selected instead of scrolling.

**Reply by arunzn** [reply]
![image](https://github.com/jlesage/docker-firefox/assets/4963459/4a086db7-ba9c-446b-9c8a-0763dd544b3d)

**Comment by jlesage** [comment]
Scrolling can be done with two fingers.  Note that even though you use a mobile browser, you are still accessing a desktop application.  Thus, the desktop application cannot behave exactly like a real mobile application.
